src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -182,6 +183,15 @@
       </IncludedFileForRunnerScript>
     </ItemGroup>
     
+    <Copy Condition="'$(TestAppX)' == 'true'"
```

Comment:
Sure, I'll create a new universal runner template for this so that it can follow this behaviour.